src/lib/components/explorer/results/AddedFilter.svelte

```diff
@@ -103,6 +112,7 @@
         on:click={editFilter}
       >
         <i class="fa-solid fa-pen-to-square"></i>
+        <span class="sr-only">Edit Filter</span>
```

Comment:
Font awesome include auto accessibility. Instead of doing this, we should set the title attribute. We can do a full pass for screen readers before we deploy to bdc. 
https://docs.fontawesome.com/web/dig-deeper/accessibility